### ISSUE TITLE
FM 508 Phase 3 Docs

### ISSUE THREAD

**Issue by cthomasONRR** [ONRR/onrr.gov-site#908]
We received 6 document from Antoinette Urioste in FM. I first need to make sure these documents are 508 compliant. Expecting some back and forth to get the docs ready to go live.

Antoinette emails Christine, Shannon, and Maroya with more documents as she gets them 508-compliant. This issue is staying open to catalog FM's remaining documents.

**Comment by cthomasONRR** [comment]
- [ ] [850710](https://app.zenhub.com/files/210892300/3b4397ea-c37a-4a9d-bc60-2a3b5299b97f/download)
- [ ] [961125](https://app.zenhub.com/files/210892300/607cd116-4bbe-48d9-bdb0-c1a4bbdc00ef/download)
- [ ] [20100607](https://app.zenhub.com/files/210892300/2323fc7a-d5ce-446a-899b-f5d9126fd4bc/download)
- [ ] [20110120](https://app.zenhub.com/files/210892300/502af6d9-4627-4bf4-920b-74338d5d04ad/download)
- [ ] [20160510](https://app.zenhub.com/files/210892300/2de19c0b-7eaa-42c5-9fd8-1c10150bf144/download)

**Comment by cthomasONRR** [comment]
I'm communicating with Antoinette Urioste via email on this project. She is working on getting these 3 PDFs 508 compliant.

**Comment by cthomasONRR** [comment]
Antoinette emailed letting us know that she needs more time for 508 remediation of these 3 documents. She is hoping to get them completed by August 10.

**Comment by cthomasONRR** [comment]
Antoinette emailed me another new PDF [930430](https://app.zenhub.com/files/210892300/de3718bf-81cd-4d0b-92c2-33edfe07e641/download) that passes 508-compliance. The updated document is now on the live site (https://www.onrr.gov/DearRep.htm). 

The doc is also updated in the accessibility tracking spreadsheet.

**Comment by cthomasONRR** [comment]
On Friday, Antoinette sent the following PDFs that pass 508-compliance:

- [ ] [850710](https://app.zenhub.com/files/210892300/5e2887c9-8052-4089-ba9b-31d52ac79d3e/download)
- [ ] [961125](https://app.zenhub.com/files/210892300/f8f89f87-4950-42c5-94d1-80b20ad3671d/download)
- [ ] [20100607](https://app.zenhub.com/files/210892300/48f3331e-9eb9-44f1-bef1-fa0d9021bb4d/download)
- [ ] [20080314](https://app.zenhub.com/files/210892300/efd5ef29-2a7a-4972-bf69-db82a02a6018/download)

All 4 PDFs pass 508-compliance, are updated on the dev and live sites, and the accessibility tracking spreadsheet is updated.

I am keeping this issue open, since Antoinette is likely to send more compliant docs.

**Comment by cthomasONRR** [comment]
Antoinette submitted another document for compliance: 1_15_16_Reporter_Letter_EFT](https://app.zenhub.com/files/210892300/7e54e078-94ad-47d8-a353-7a4668ab74cf/download).

It still had a error (table summary - table was unnecessary formatting for the text it highlighted, so I deleted it and the error resolved), but I fixed it and put the new document online. I also updated the accessibility tracking spreadhseet.

**Comment by cthomasONRR** [comment]
Antoinette submitted 4 more documents. They are on the live site and the accessibility tracking spreadsheet is updated.

- [ ] [920514](https://app.zenhub.com/files/210892300/6ad09974-454c-4f3f-b0a6-c1426c5b54ee/download)
- [ ] [950925](https://app.zenhub.com/files/210892300/7081de5d-c12d-4542-83d5-eadf3d06b473/download)
- [ ] [20070511](https://app.zenhub.com/files/210892300/34e7ea59-2714-4dec-ac3b-0a2a2ceebad6/download)
- [ ] [20070927](https://app.zenhub.com/files/210892300/a8973da8-0424-4500-acea-e7de778f0d49/download)

**Comment by cthomasONRR** [comment]
Antoinette submitted 3 more documents. They are on the live site and the accessibility tracking spreadsheet is updated.

- [ ] [20000405](https://app.zenhub.com/files/210892300/aa517dda-abab-4fdd-b981-b792e8419e11/download)
- [ ] [20070723](https://app.zenhub.com/files/210892300/12102d51-bd40-4f28-883b-0828c89fc3f6/download)
- [ ] [20070911](https://app.zenhub.com/files/210892300/70e4b12e-ff82-4689-b5ac-bf9974e6cb3f/download)

**Comment by cthomasONRR** [comment]
Antoinette submitted 3 more documents. They are on the live site and the accessibility tracking spreadsheet is updated.

- [ ] [20140630.pdf](https://github.com/ONRR/onrr.gov-site/files/7180002/20140630.pdf)
- [ ] [970331.pdf](https://github.com/ONRR/onrr.gov-site/files/7180004/970331.pdf)
- [ ] [971001.pdf](https://github.com/ONRR/onrr.gov-site/files/7180005/971001.pdf)

**Comment by cthomasONRR** [comment]
Antoinette sent 3 more remediated PDFs to upload:

- [ ] [920326](https://app.zenhub.com/files/210892300/b6f6d21f-507d-44e5-b233-62c7885e6808/download)
- [ ] [900420](https://app.zenhub.com/files/210892300/2ce53d70-104a-4c91-b174-6780997e4acc/download)
- [ ] [920206a](https://app.zenhub.com/files/210892300/d8c15a99-37b3-4772-a9b4-f5aad091de0b/download)

**Comment by cthomasONRR** [comment]
Antoinette sent 3 more remediated PDFs to upload. They are on the live site and the accessibility tracking spreadsheet is updated.

- [ ] [950929.pdf](https://github.com/ONRR/onrr.gov-site/files/7554965/950929.pdf)
- [ ] [951004.pdf](https://github.com/ONRR/onrr.gov-site/files/7554966/951004.pdf)
- [ ] [20020320.pdf](https://github.com/ONRR/onrr.gov-site/files/7554967/20020320.pdf)

**Comment by cthomasONRR** [comment]
Antoinette sent 4 more remediated PDFs to upload. They are on the live site and the accessibility tracking spreadsheet is updated.

- [ ] [900913.pdf](https://github.com/ONRR/onrr.gov-site/files/7557916/900913.pdf)
- [ ] [930900a.pdf](https://github.com/ONRR/onrr.gov-site/files/7557917/930900a.pdf)
- [ ] [930900b.pdf](https://github.com/ONRR/onrr.gov-site/files/7557918/930900b.pdf)
- [ ] [20041116.pdf](https://github.com/ONRR/onrr.gov-site/files/7557919/20041116.pdf)

**Comment by cthomasONRR** [comment]
Antoinette sent 1 more remediated PDF to upload: 
- [ ] 20040220.pdf
This document is on the live site and the accessibility tracking spreadsheet is updated.

She said that another PDF:
- [ ] 20-CFR-1243
Cannot be edited as it is from the CFR (ONRR does not own it). I asked her if we can remove this PDF and instead link to the eCFR location (https://www.ecfr.gov/current/title-30/chapter-XII/subchapter-A/part-1243?toc=1). Waiting for her response.

**Comment by cthomasONRR** [comment]
Antoinette gave her OK to remove 30-CFR-1243. I removed the document, replaced the document with the link, and updated the accessibility tracking spreadsheet.

**Comment by cthomasONRR** [comment]
Antoinette re-sent the final documents. She had corrected them in July and they were loaded in the Y and Z drives, but they hadn't been marked on the accessibility tracking sheet. Everything (including the accessibility tracking sheet) is now updated. FM is officially complete! I can finally close this issue!!!
